packages/runtime/src/index.js

```
@@ -0,0 +1,47 @@
+        }
+        document.body.appendChild(elemento);
+      }catch(error){
+        console.error(error);
+      }
+      if(etiquetas[i].hasAttribute("src")){
+        try{
+          fetch(etiquetas[i].getAttribute("src")).then(res => res.text()).then(data => {
+            let codigoCo2m = esJscore.compile(data);
+            let element2o = document.createElement("script");
+            element2o.innerHTML = codigoCo2m;
+            if(estiquetas[i].hasAttribute("mode") && etiquetas[i].getAttribute("mode") == "modulo"){
```

Biome

[error] Use === instead of ==. == is only allowed when comparing against `null`

== is only allowed when comparing against null

Using == may be unsafe if you are relying on type coercion
Unsafe fix: Use ===


(lint/suspicious/noDoubleEquals)